src/Sylius/Bundle/UiBundle/Resources/private/js/sylius-user.js

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
This file should be removed, because it is duplicated. Please use a file from SyliusUserBundle.